content/media-player/tracking/setup.md

```diff
@@ -0,0 +1,767 @@
+> 4. V2 schemas support live streaming video (in addition to VOD).
+
+| Tracking SDK | Platform | Media schemas version | Provides auto-tracking? | Player |
+| --- | --- | --- | --- | --- |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/media/ | Web | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/vimeo-tracking/ | Web  | v2 | Yes | Vimeo |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (iOS) | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (iOS) | v2 | Yes | AVPlayer |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/media-tracking/ | Mobile (Android) | v2 | No | Any |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/media-tracking/ | Web | v1 | Yes | HTML5 media or video |
+| https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/youtube-tracking/ | Web | v1 | Yes | YouTube |
+
+#### **Step 2:** Install the tracking SDK in your app
+
+Once you have chosen the tracking SDK to use, the next step is to install it in your app.
+
+{{< tabs groupId="tracking sdk" >}}
+
+{{% tab name="Media JS plugin" %}}
+
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-media` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-media
+```
+{{% /tab %}}
+
+{{% tab name="Vimeo JS plugin" %}}
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-vimeo-tracking` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-vimeo-tracking
+```
+{{% /tab %}}
+
+{{% tab name="YouTube JS plugin" %}}
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-youtube-tracking` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-youtube-tracking
+```
+{{% /tab %}}
+
+{{% tab name="HTML5 media tracking JS plugin" %}}
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-media-tracking` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-media-tracking
+```
+{{% /tab %}}
+
+{{% tab name="Web – script tag" %}}
```

Comment:
I think we can remove this tab, the number of options must be quite confusing for users and this doesn't really fit with the other ones